spark/spark-tensorflow-connector/README.md

```diff
@@ -63,6 +63,30 @@ When writing Spark DataFrame to TensorFlow records, the API accepts several opti
 * `recordType`: output format of TensorFlow records. By default it is Example. Possible values are:
   * `Example`: TensorFlow [Example](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/example/example.proto) records
   * `SequenceExample`: TensorFlow [SequenceExample](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/example/example.proto) records
+* `writeLocality`: determines whether the TensorFlow records are written locally on the workers
+or on a distributed file system. Possible values are:
+  * `distributed` (default): the dataframe is written using Spark's default file system.
+  * `local`: writes the content on the disks of each the Spark workers, in a partitioned manner
+  (see details in the paragraph below).
+
+_Local mode write_ each of the workers stores on the local disk a subset of the data.
+Which subset that it stored on each worker is determined by the partitioning of the Dataframe.
```

Comment:
Change to "The subset that is stored ..."

Reply:
Done